v3/newrelic/segments.go

```diff
@@ -337,6 +340,30 @@ func StartExternalSegment(txn *Transaction, request *http.Request) *ExternalSegm
 	return s
 }
 
+func StartExternalSegmentFastHTTP(txn *Transaction, ctx *fasthttp.RequestCtx) *ExternalSegment {
+	if nil == txn {
+		txn = transactionFromRequestContextFastHTTP(ctx)
+	}
+	request := &http.Request{}
+
+	fasthttpadaptor.ConvertRequest(ctx, request, true)
+	s := &ExternalSegment{
+		StartTime: txn.StartSegmentNow(),
+		Request:   request,
+	}
+	s.secureAgentEvent = secureAgent.SendEvent("OUTBOUND", request)
```

Comment:
This statement should be inside an `if IsSecurityAgentPresent(){ ... }` conditional